index.html

```diff
@@ -37,7 +37,172 @@ <h1 >I’m Esmatullah Glad to see you!</h1>
+                    </div>
+                    <div class="project-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul>
+                            <li class="porject-tect-btn">html</li>
+                            <li class="porject-tect-btn">css</li>
+                            <li class="porject-tect-btn">javaScript</li>
+                        </ul>
+                    </div>
+                    <button class="regular-btn">See Project</button> 
+                </div>
+            </div>
+    
+            <div class="projects">
+                <div class="project-img">
+                    <img class="tonic-project-img" src="images/fourth project.svg" alt="">
+                </div>
+                <div class="project-info">
+                    <div class="project-name">
+                        <h2>Multi-PostStories</h2>
+                    </div>
+                    <div class="project-subtitle">        
+                        <p>CANOPY</p>
+                        <span></span>
+                        <p>Back End Dev</p>
+                        <span></span>
+                        <p>2015</p>
+                    </div>
+                    <div class="project-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul>
+                            <li class="porject-tect-btn">html</li>
+                            <li class="porject-tect-btn">css</li>
+                            <li class="porject-tect-btn">javaScript</li>
+                        </ul>
+                    </div>
+                    <button class="regular-btn">See Project</button> 
+                </div>
+            </div>
+        </section>
+
+        <section class="primary-font about-me">
+            <div class="">
+                <h1 class="about-title">About Myself</h1>
+                <p class="about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button class="regular-btn">Get my resume</button> 
+                <div class="social-media-link">
+                    <ul class="contact-list">
+                        <li><a href=""><img src="icons/twitter.svg" alt="twitter icon"></a></li>
+                        <li><a href=""><img src="icons/linked.svg" alt="linkedin icon"></a></li>
+                        <li><a href=""><img src="icons/Vector1.svg" alt="social media icon"></a></li>
+                        <li><a href=""><img src="icons/github.svg" alt="github icon"></a></li>
+                        <li><a href=""><img src="icons/Vector2.svg" alt="social media icon"></a></li>
+                 =   </ul>
```

Comment:
Kindly remove the equal mark as it shows up in the page and add valid social account links to redirect user to user accounts